dream_layer_backend/dream_layer.py

```
@@ -567,6 +567,360 @@ def get_controlnet_models_endpoint():
             "message": f"Failed to fetch ControlNet models: {str(e)}"
         }), 500
 
+@app.route('/api/create-labeled-grid', methods=['POST'])
+def create_labeled_grid():
+    """Create a labeled grid from images with enhanced features"""
+    try:
+        from dream_layer_backend_utils.labeled_grid_exporter import (
+            assemble_grid_enhanced, collect_images, read_metadata, 
```

Ruff (0.12.2)

`dream_layer_backend_utils.labeled_grid_exporter.collect_images` imported but unused

Remove unused import

(F401)